src/ffi/c.rs

```diff
@@ -21,7 +20,7 @@ impl ErrorMessage {
 }
 
 pub struct StreamWrapper {
-    pub inner: Box<mz_stream>,
+    pub inner: *mut mz_stream,
```

Comment:
Since this field is a pointer now, could we document everything we need to know about it here?